php56/provision.sh

```diff
@@ -0,0 +1,164 @@
+#!/usr/bin/env bash
+
+DIR=`dirname $0`
+
+# PACKAGE INSTALLATION
+#
+# Build a bash array to pass all of the packages we want to install to a single
+# apt-get command. This avoids doing all the leg work each time a package is
+# set to install. It also allows us to easily comment out or add single
+# packages. We set the array as empty to begin with so that we can append
+# individual packages to it as required.
+apt_package_install_list=()
+
+# Start with a bash array containing all packages we want to install in the
+# virtual machine. We'll then loop through each of these and check individual
+# status before adding them to the apt_package_install_list array.
+apt_package_check_list=(
+
+  # PHP5.6
+  #
+  # Our base packages for php5.6. As long as php5.6-fpm and php5.6-cli are
+  # installed, there is no need to install the general php5.6 package, which
+  # can sometimes install apache as a requirement.
+  php5.6-fpm
+  php5.6-cli
+
+  # Common and dev packages for php
+  php5.6-common
+  php5.6-dev
+
+  # Extra PHP modules that we find useful
+  php5.6-imagick
+  php5.6-memcache
+  php5.6-bcmath
+  php5.6-curl
+  php5.6-gd
+  php5.6-mbstring
+  php5.6-mcrypt
+  php5.6-mysql
+  php5.6-imap
+  php5.6-json
+  php5.6-soap
+  php5.6-ssh2
+  php5.6-xdebug
+  php5.6-xml
+  php5.6-zip
+)
+
+### FUNCTIONS
+
+network_detection() {
```

Comment:
I have an idea how we can, but i would rather get the feature in then do that in a separate pr when i solve it in VVV core.